src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.GraphQL.approved.txt

```diff
@@ -1882,7 +1878,10 @@ namespace GraphQL.Types
         GraphQL.Types.IObjectGraphType Mutation { get; set; }
         GraphQL.Conversion.INameConverter NameConverter { get; set; }
         GraphQL.Types.IObjectGraphType Query { get; set; }
+        GraphQL.Types.FieldType SchemaMetaFieldType { get; }
```

Comment:
By the way this is a breaking change for custom schemas (if any).

Reply:
If it gets pulled, I'll put it in the upgrade notes

Reply:
Of course it will be pulled.